run_plotlc.py

```diff
@@ -136,6 +146,8 @@ def annotate(sel):
 	mplcursors.cursor(ax).connect("add", annotate)
 	plt.show(block=True)
 
+	if args.savephot != 'None':
```

Comment:
Think we should do this in another PR... It has nothing to do with zeropoints and bootstrapping... And it is producing another data product, which I think we should discuss...

Reply:
This is now fixed with this [commit](https://github.com/rhandberg/flows/pull/10/commits/7bde92818dac5058cc071102d5b65464a2913623).

I'll add a separate branch for these changes. I reverted back to the last change made by you.